README.md

```diff
@@ -1 +1,24 @@
-# Hello-world
\ No newline at end of file
+Hello-world is a simple project to help me to get familiar with the basic tools I needed to learn and put into practice in this project.
+
+Built With
+Client
+. html
+. CSS
+
+Key Features
+. Displaying header with a red text "Hello Microverse!"
+
+Getting Started
+To get a local copy up and running, follow these steps.
+Prerequisites
+In order to run this project you need:
+. git terminal and github link
+
+Setup
+Clone this repository to your desired folder:
+. git clone https://github.com/peteryeungtt/Hello-world.git
+
+Author1
+GitHub: peteryeungtt
+Twitter: @PeterYeungJW
```

Comment:
- [ ] Kindly use the `microverse` provided template because the template that you are using is not professional. Microverse templates have everything to showcase your abilities, skills etc, in terms of technical knowledge. You just need to customize that template according to your project requirement. Take a look at the below screenshot for more detail.

![Screenshot from 2023-04-25 17-02-50](https://user-images.githubusercontent.com/66476329/234264454-3abd71a9-83ee-4c73-8cfb-c656d36b6e9f.png)

- Please use the provided template and the link for the template is [here](https://github.com/microverseinc/readme-template)

Reply:
Thank you again, I adjusted that also

Reply:
@skyv26 Adjusted thank you so much for your feedback